model/client4.go

```diff
@@ -1794,6 +1794,18 @@ func (c *Client4) PermanentDeleteTeam(teamId string) (bool, *Response) {
 	return CheckStatusOK(r), BuildResponse(r)
 }
 
+// UpdateTeamPrivacy modifies the team type (model.TEAM_OPEN <--> model.TEAM_INVITE) and sets
+// the corresponding AllowOpenInvite appropriately.
+func (c *Client4) UpdateTeamPrivacy(teamId string, privacy string) *Response {
+	query := fmt.Sprintf("?privacy=%s", privacy)
```

Comment:
In case we move it to the body, change this

Reply:
Done, thanks.  https://github.com/mattermost/mattermost-server/pull/14287/commits/045457a23d4112cf390b6ae5f244a0ae194f1ec8